en/FlyView/FlyView.md

```diff
@@ -1,79 +1,375 @@
+- Control missions: [start](#start_mission), [continue](#continue_mission), [pause](#pause), [stop](#stop_mission) and [resume](#resume_mission).
+- Guide the vehicle to [arm](#arm)/[disarm](#disarm)/[emergency stop](#emergency_stop), [takeoff](#takeoff)/[land](#land), [change altitude](#change_altitude), [go to](#goto) or [orbit](#orbit) a particular location, and [return/RTL](#rtl).
+- Switch between a map view and a video view (if available)
+- Display video, mission, telemetry, and other information for the current vehicle, and also switch between connected vehicles.
 
-## Map
+![Fly View](../../assets/fly/fly_view_overview.jpg)
 
-The map will show you the positions of all the vehicles you are connected to.
-It will also show you the mission for the current vehicle.
+## UI Overview
 
-## Fly Tools
+The screenshot above shows the main elements of the fly view:
 
-On the left edge of the screen you will see the Fly Tools. The order of tools from top to bottom is:
+- **Map:** Displays the positions of all connected vehicles and the mission for the current vehicle.
+  - You can drag the map to move it around (the map automatically re-centres after a certain amount of time).
+  - Once flying, you can click on the map to set a [Go to](#goto) or [Orbit at](#orbit) location.
+- **Fly Toolbar:** Key status information for sensors (GPS, battery, RC control), and vehicle state (Flight mode, Armed/Disarmed status).
+  - Select the sensor indicators to view more detail.
+  - Click the *Flight mode* text (e.g. "Hold") to select a new mode. Not every mode may be available.
+  - Click the *Armed/Disarmed* text to toggle the armed state. If flying you can click this to Emergency Stop.
+- **Fly tools:** You can use these to:
+  - Toggle between takeoff/land.
+  - Pause/restart the current operation (e.g. landing, or the mission).
+  - Safety return (also known as RTL or Return).
+  - The *Action* button offers other appropriate options for the current state (these overlay the *Confirmation Slider*). 
+    Actions include changing the altitude or continuing a mission.
+  - Enable the [preflight checklist](#preflight_checklist) (tool option disabled by default).
+- **[Instrument Panel](#instrument_pane):** A multi-page widget that displays vehicle information including: telemetry, camera, video, system health, and vibration.
+- **[Video/Switcher](#video_switcher):** Toggle between video or map in a window.
+  - Click the element to switch *Video* and *Map* to foreground.
+  - *QGroundControl* supports RTP and RTSP video streaming over your vehicles UDP connection. 
+    It also support directly connected UVC device support.
+    QGC video support is further discussed in the [Video README](https://github.com/mavlink/qgroundcontrol/blob/master/src/VideoStreaming/README.md).
+- **Confirmation Slider:** Context sensitive slider to confirm requested actions. 
+  Slide to start operation. Press **X** to cancel.
```

Comment:
```suggestion
  Slide to start operation. Click**X** to cancel.
```

Reply:
It's not a click if it's on mobile is it?

Reply:
"Press" actually made me think of pressing the "X" key in my keyboard for a few seconds.
"Press the **X** Icon to cancel" would be fine too.
But this page currently has 24 mentions of "click" and only this one of "press".

Reply:
The whole library uses click because I test and write it on a desktop. Press is more generic, so I will change this document, and try to change other cases as I go along.